build/de.cau.cs.kieler.klighd.repository/pom.xml

```diff
@@ -46,14 +52,14 @@
             <configuration>
               <associateSites>
                 <!-- Eclipse -->
-                <associateSite>https://download.eclipse.org/releases/2020-06/</associateSite>
+                <associateSite>https://download.eclipse.org/releases/2021-06/</associateSite>
                 <!-- UML2 Tools (Meta Model Implemenetation, ...) -->
                 <associateSite>https://download.eclipse.org/modeling/mdt/uml2/updates/5.4/</associateSite>
                 <!-- Eclipse Layout Kernel -->
                 <!-- associateSite>http://build.eclipse.org/modeling/elk/updates/nightly/</associateSite -->
                 <associateSite>https://download.eclipse.org/elk/updates/releases/0.8.1/</associateSite>>
                 <!-- Xtext -->
-                <associateSite>https://download.eclipse.org/modeling/tmf/xtext/updates/releases/2.22.0/</associateSite>>
+                <associateSite>https://download.eclipse.org/modeling/tmf/xtext/updates/releases/2.25.0/</associateSite>>
                 <associateSite>https://xtext.github.io/download/updates/releases/2.1.1/</associateSite>>
                 <!-- Eclipse Orbit for Google Guave, Apache Batik, ... -->
                 <associateSite>https://download.eclipse.org/tools/orbit/downloads/drops/R20200529191137/repository/</associateSite>
```

Comment:
Why does this update the Eclipse release, but not its associate orbit release? There might appear inconsistencies because of this.

Reply:
Right, I overlooked that.

Reply:
switched to orbit release R20210223232630 instead of R20210602031627, since R20210602031627 brings 'com.google.inject:5.0.1' that I wanna exclude for now